.github/workflows/e2e-tests.yml

```
@@ -36,8 +44,11 @@ jobs:
     name: iOS E2E Tests
     runs-on: macos-15
     timeout-minutes: 120
-    # Skip on forked PRs — secrets are not available in forks
-    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+    # Skip on forked PRs and when E2E secrets are not configured
+    if: >
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)